src/common/components/selector-input-list.tsx

```diff
@@ -146,14 +146,14 @@ export class SelectorInputList extends React.Component<SelectorInputListProps, S
     }
 
     @autobind
-    private getDeleteSelectorHandler(item: string[]) {
+    private getDeleteSelectorHandler(item: string[]): (event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void {
```

Comment:
do we need the `MouseEvent` for the type?

Reply:
Updated.